converter/utils.py

```diff
@@ -0,0 +1,18 @@
+# -*- encoding: utf-8 -*-
+"""
+This file contains supporting logic
+
+1. is_float (Check if the user's input convertible to float type)
+"""
+from typing import Any
+
+
+def is_float(arg: Any) -> bool:
```

Comment:
@MariaMaximova, can you add unit test for this function?

Reply:
Done. [Tests were added. Branch issue-5](https://github.com/MariaMaximova/convert_temp/pull/9/commits/eca242da1d03d94023b13adb005e0fc96bafc7ec)